szachy.html

```diff
@@ -8,8 +8,8 @@
 
 
 <div class="chessboard-container">
-    <div id="chessboard-automated" class="chessboard">
-    </div>
+    <div class="wrapper"></div>
```

Comment:
what about this empty wrapper div? it would be nice to try implement this that way (as i've mention in prev comment)